spec/controllers/api/v1/offers_controller_spec.rb

```diff
@@ -469,6 +469,41 @@ def epoch_ms(time)
       end
     end
 
+    context "filter by type" do
+      before(:each) {
+        schedule
+        schedule1
+        delivery
+        delivery1
+        scheduled_offer
+        scheduled_offer1
+      }
+
+      it "can return offers in descending order if 'sort_type' is 'created_at_desc' in params" do
+        get :search, column: 'created_at', is_desc: true
+        expect(response.status).to eq(200)
+        expect(subject['offers'].first["id"]).to eq(scheduled_offer1.attributes["id"])
```

Comment:
given that it's a sort, I'd suggest testing that all the returned offers are in the correct order rather than just testing the first one